src/broker/cas_cgw.c

```diff
@@ -2090,3 +2103,22 @@ cgw_is_support_datatype (SQLSMALLINT data_type, SQLLEN type_size)
 
   return support_data_type;
 }
+
+static int
+cgw_count_number_of_digits (int num_bits)
+{
+  int num = 1;
+  INT64 number = 1;
+
+  if (num_bits <= 0)
```

Comment:
What about check upper limit also to avoid possible overflow, for example:
`if (num_bits <= 0 || num_bits > 64)`

Reply:
fixed it